### PR TITLE
[11.0] reservation summary click actions

### PR DESCRIPTION
**Pull request by robinkeunen**
# DO NOT MERGE BEFORE #113 
In the reservation summary

- clicking on the room cell opens the room form
- clicking on a busy cell opens the reservation form

### CODE REVIEW

**Comment by robinkeunen** [comment]
# DO NOT MERGE BEFORE #113

File: hotel_reservation_summary_base/models/room.py [hunk 1]
```diff
@@ -46,11 +46,20 @@ def get_room_daily_summary(self, day):
         room_status, reservation_lines = self.get_room_status(day)
         room_status_text = reservation_lines.get_room_summary_cell_text(day)
 
+        if reservation_lines:
```
Comment (mclaeysb) [inline hotel_reservation_summary_base/models/room.py:55]: Seems right!

**Review by mclaeysb — COMMENTED** [review]
Look good!

**Comment by mclaeysb** [comment]
CI seems to pass now, but there still an error in one of the travis checks ... that seems to be unrelated to this code?

**Comment by robinkeunen** [comment]
Looks like the CI error, maybe try to push a dummy fixup commit to trigger the CI?

**Comment by mclaeysb** [comment]
Many thanks for fixing and making it green again!

**Pull request closed without merging**